pkg/cmd/repo/credits/credits.go

```diff
@@ -61,6 +61,9 @@ func NewCmdCredits(f *cmdutil.Factory, runF func(*CreditsOptions) error) *cobra.
 			return creditsRun(opts)
 		},
 		Hidden: true,
+		Annotations: map[string]string{
```

Comment:
i would think this wouldn't be needed for `Hidden` commands, right? shouldn't those just automatically not have markdown generated for them?

Reply:
That is a good question. I opted to remove the logic 
```
!c.IsAvailableCommand() || c.IsAdditionalHelpTopicCommand()
```
from markdown generation in favor of our own custom annotation. The reason being is that I felt relying on Cobra's functions that check `Hidden: true` was a bit opaque as well as overloaded, in the sense that it not only stops displaying the command in `gh` but it also stops doc generation. That logic specifically does not work for the help topics where we want the command to be hidden but docs to still be generated. 

The other direction I considered was adding a specific annotation to just the help topic commands that would force generation of the docs. I felt the implemented direction was more consistent and further removed Cobra logic.

Reply:
I think the least confusing thing for now is to consult Hidden when deciding whether or not to generate markdown and still rely on the new annotation for non-hidden commands. Even if we quit cobra tomorrow we'd still want to support a notion of `Hidden` commands and they'd still automatically not get markdown generation.

Reply:
I changed the logic so that hidden commands won't have markdown generated unless they specify the `markdown:generate` annotation.